spec/lib/local-links-manager/check_links/link_status_requester_spec.rb

```diff
@@ -8,20 +8,26 @@
   subject(:link_status_requester) { described_class.new }
 
   context "links for enabled services" do
```

Comment:
I can see why this doesn't say `homepage_urls`, as the difference between this context and the next is that the services are enabled or disabled.  Maybe we should name one `with homepage urls and links for enabled services` and the other `with homepage urls and links for disabled services`?